src/core/glob_matcher.h

```diff
@@ -10,16 +10,15 @@
 
 // We opt for using Reflex library for glob matching.
 // While I find PCRE2 faster, it's not substantially faster to justify the shared lib dependency.
-#define REFLEX_PERFORMANCE
+//#define REFLEX_PERFORMANCE
```

Comment:
please add explanation why it was disable and the reference to this PR for more details

Reply:
Yes I will!